### DISCUSSION TITLE
Proxy serveur IP banned from Vercel

### DISCUSSION THREAD

**Discussion by ipatate** [vercel/community#294] [Help] [answered]
Hello, we use a proxy server before vercel for redirecting to the good app. The domain is shared between multiple site. One of this site, is a Next JS on vercel. We use this methode https://nextjs.org/blog/incremental-adoption#subpath
Yesterday, our IP is banned from vercel and the website is down.
How correct this ? It's possible to whitelist IP ? thanks

**Comment by gogocharli** [comment, marked as answer]
Hello there, could you [create a case](https://vercel.com/support/request) and communicate your proxy server's address alongside your case number. We will take a look.

**Reply by ipatate** [reply]
thanks, it's done, but the support is long for respond. it's minimum 48h, it's long when your site is down

**Reply by gogocharli** [reply]
@ipatate Could you communicate your case number? It will start with **000**

**Reply by ipatate** [reply]
we have received a response today but not the solution for our problem. We try netlify for use instead of vercel. The same proxy work on netlify and not on vercel since 1 week now. Why ? disapointed by the situation :(
